### PR TITLE
Bugfix #959 & #832

### PR DESCRIPTION
**Pull request by JulianWielga**
Bugfix for #959 & #832

### CODE REVIEW

File: ui/client/containers/Visualization.js [hunk 1]
```diff
@@ -58,7 +58,12 @@ class Visualization extends React.Component {
 
   bindPasteShortcut() {
     return (event) => this.props.actions.pasteSelection(
-      () => this.pasteSelection(event),
+      () => {
+        if (["INPUT", "TEXTAREA"].includes(event.target?.tagName)) {
```
Comment (mproch) [inline ui/client/containers/Visualization.js:62]: does this mean if I copy nodes and "paste" them to textarea I'll get json?

File: ui/client/components/graph/focusable.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React, {forwardRef, useCallback, MouseEventHandler} from "react"
```
Comment (mproch) [inline ui/client/components/graph/focusable.tsx:1]: What is it for? To be able to select sth on graph pane?

File: ui/client/containers/Visualization.js (hunk 1 above)
Comment (JulianWielga) [reply ui/client/containers/Visualization.js:62]: yes. if user has focus on input field then he wants to fill this input. pasting clipboard content somewhere else is confusing

File: ui/client/components/graph/focusable.tsx (hunk 2 above)
Comment (JulianWielga) [reply ui/client/components/graph/focusable.tsx:1]: to grab focus e.g. from comment input when clicked on graph

**Pull request merged**